apps/admin-x-activitypub/src/components/search/ProfileSearchResultModal.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import NiceModal, {useModal} from '@ebay/nice-modal-react';
+import {ActorProperties} from '@tryghost/admin-x-framework/api/activitypub';
+import {Button, Heading, Modal} from '@tryghost/admin-x-design-system';
+
+import APAvatar from '../global/APAvatar';
+import FollowButton from '../global/FollowButton';
+
+interface ProfileSearchResultModalProps {
+    profile: {
+        actor: ActorProperties;
+        handle: string;
+        isFollowing: boolean;
+    };
+}
+
+const ProfileSearchResultModal: React.FC<ProfileSearchResultModalProps> = ({profile}) => {
+    const modal = useModal();
+    const attachments = (profile.actor.attachment || [])
+        .filter(attachment => attachment.type === 'PropertyValue');
+
+    return (
+        <Modal
+            align='right'
+            animate={true}
+            footer={<></>}
+            height={'full'}
+            padding={false}
+            size='bleed'
+            width={640}
+        >
+            <div className='sticky top-0 z-50 border-grey-200 bg-white py-3'>
+                <div className='grid h-8 grid-cols-3'>
+                    <div className='col-[3/4] flex items-center justify-end space-x-6 px-8'>
+                        <Button icon='close' size='sm' unstyled onClick={() => modal.remove()}/>
+                    </div>
+                </div>
+            </div>
+            <div className='mt-10 w-auto'>
+                <div className='mx-auto max-w-[580px] pb-16'>
+                    <div className='z-0 flex w-full flex-col items-center'>
+                        <div className='mx-auto w-full max-w-[560px]'>
+                            <div className='h-[200px] w-full rounded-lg bg-gradient-to-tr from-grey-200 to-grey-100'>
```

Comment:
@djordjevlais There might be a few too many divs nested here 😅 Think this can probably be refactored